src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+import UI.UserInterface;
+import tasks.tasktypes.DeadlineTask;
+import tasks.tasktypes.EventTask;
+import tasks.tasktypes.ToDoTask;
+import tasks.TaskList;
+
 public class Duke {
+    /** Command list to check against */
+    private static final String INPUT_TERMINATE = "bye";
+    private static final String INPUT_LIST = "list";
+    private static final String INPUT_MARK = "mark";
+    private static final String INPUT_UNMARK = "unmark";
+    public static final String INPUT_DEADLINE = "deadline";
+    public static final String INPUT_EVENT = "event";
+
+    /** tasks.TaskList class that contains all items */
+    private static final TaskList TASK_LIST = new TaskList();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greet();
+        programBody();
+        exit();
+    }
+
+    /**
+     * Body of the program. Runs in a loop until terminated.
+     */
+    public static void programBody() {
+        while (true) {
+            String input = UserInterface.getInput();
+            if (isTerminatingInput(input)) {
```

Comment:
Perhaps you could consider switch-case? I have seen other coder who used switch-case for better readability. 
However this is a very minor issue.